glide.lock

```diff
@@ -38,13 +48,13 @@ imports:
   - design
   - design/apidsl
   - dslengine
+  - encoding/form
   - goagen
   - goagen/codegen
   - goagen/gen_app
   - goagen/gen_controller
   - goagen/utils
   - goatest
-  - logging/logrus
```

Comment:
do we really want to remove logrus?

Reply:
we can use the logger wrapper provided by `fabric8-auth`